wfe/wfe.go

```diff
@@ -238,13 +246,86 @@ func (wfe *WebFrontEndImpl) RootCert(
 	_, _ = response.Write(root.PEM())
 }
 
+func (wfe *WebFrontEndImpl) RootKey(
+	ctx context.Context,
+	response http.ResponseWriter,
+	request *http.Request) {
+
+	key := wfe.ca.GetRootKey()
+	if key == nil {
+		response.WriteHeader(http.StatusServiceUnavailable)
+		return
+	}
+
+	var buf bytes.Buffer
+
+	err := pem.Encode(&buf, &pem.Block{
+		Type:  "RSA PRIVATE KEY",
+		Bytes: x509.MarshalPKCS1PrivateKey(key),
+	})
+	if err != nil {
+		panic(fmt.Sprintf("Unable to encode private key to PEM: %s",
```

Comment:
I think it might be better to replace the `panic` with `wfe.sendError(acme.InternalErrorProblem(msg)); return`. That's the way we've handled other "shouldn't happen" conditions (e.g. see `Directory` when `relativeDirectory` returns an err).

Reply:
Done.